docs/data-management.rst

```diff
@@ -130,6 +132,33 @@ Browsing and rendering
 
 #. You can revert to the original settings for an Image or Dataset. For example, using the context menu for a Dataset in the tree, select ``Rendering Settings > Set Imported and Save``.
 
+
+Manage Images in Dataset/Projects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can organize the data in the left-hand side tree by creating new Projects and Datasets. You can link the Images to the new or existing Datasets and Datasets to new or existing Projects.
+
+#. Use the Project |image2b| and Dataset |image2c| icons above the left-hand side tree or the right-click contextual menu to create new Datasets or Projects.
+
+#. Drag and drop Images between Datasets and Datasets between Projects.
+
+#. Copy Images using the right-click context menu: 
+     - Select the Images to be copied, then right-click and click ``Edit > Copy Link``
+     - Select the Dataset you want to copy the Images to, right-click and click on ``Edit > Paste Link``
+
+       |image2d|
+
+.. warning::
+    **The ``Copy Link`` feature will not create a new independent copy of the Images.** There is no possibility to copy Images in OMERO at the moment in a way resulting in a new independent copy. The only thing that is copied during the actions above is the link between the Images and the Dataset. A single Image becomes linked to two Datasets.
+    
+    If you delete one of the Datasets, any Images within it that are linked to other Datasets will be retained. Nevertheless, if **you directly select and delete an Image that has been copied from another Dataset it will be deleted and lost from both Datasets.** There is a clear warning in the OMERO.web when you try to delete such doubly-linked Image, see screenshot below.
+
+
+|image2e|
+
+.. note::
+    If you are an administrator or administrator with restricted privileges working in a group you are not a member of: All containers (Projects, Datasets, Screens) created in OMERO.web in such group will belong to the user whose portfolio you are working with. This helps to retain the possibility for this user to manipulate their containers even though you created them. Nevertheless if the administrator creates new links between such containers and other objects of the user, these will belong to the administrator and the user will not be able to unlink the objects later.
```

Comment:
``creates new links between such containers and other objects of the user, these will belong to the administrator and the user will not be able to unlink the objects later.``
indicate that it is a current limitation. cf. GH issue

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/5/commits/592c43f536e4835707f593d062df3b1d016f1ff7